pages/index.vue

```diff
@@ -1,16 +1,10 @@
 <template>
```

Comment:
Can we remove component files too?

Reply:
No for now we can't remove the component files as they have been reused in other pages as well, all of them, like OurInvestors are included in About us, Community is included in Technology.